pkg/bootstrap/bootstrap_config.go

```diff
@@ -15,10 +15,13 @@
 package bootstrap
```

Comment:
Maybe just call this file config? Its already in the bootstrap package so is currently stuttering.

Reply:
Also this seems like a weird directory to have pilot (agent) code? Shouldn't this be somewhere like `pilot/pkg/bootstrap/agent`?

Reply:
Rename: I agree, but not a priority now. When we deprecate v1 we'll refactor/split this.

I'm trying to remove 'agent' from pilot - the code may be merged with node_agent (from security), 
and it doesn't have many dependencies on pilot (or even envoy - technically can start any process with a template).